packages/components/image/__tests__/image.test.tsx

```
@@ -4,9 +4,39 @@ import {render, act} from "@testing-library/react";
+          this._onload = fn;
+        },
+        configurable: true,
+      });
+    }
+
+    trackImageOnload();
+  });
+
+  afterAll(() => {
+    // Restore original Image prototype
+    delete window.Image.prototype._onload;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)